build/cockroach-template.rb

```diff
@@ -16,29 +16,22 @@ def install
     ENV["GOPATH"] = buildpath
     ENV["GOHOME"] = buildpath
 
-    files = Dir.glob("*")
+    # TODO(peter): It would be nicer to do something like:
+    #     git ls-tree --name-only HEAD
+    files = Dir.glob("*") + Dir.glob(".{git*,dockerignore,editorconfig}")
```

Comment:
There is also a `.brew_home` directory that I was trying to avoid moving, but it appears that doesn't matter. I also googled enough ruby to figure out how to delete `.brew_home` from the `files` array.